samples/gradle-android/README.md

```diff
@@ -95,3 +95,7 @@ Note: it doesn’t work very well with Unit tests and it’s actually flaky at t
 ```
 $ ./gradlew :app:assembleDebug -Pkotlin.incremental=true
 ```
+
+## Pro Tips
```

Comment:
Does not fit the Performance Optimizations section. Also, `Pro` is as lame as new MacBooks (RIP).

```
# Tips
## Auto-download Android SDK
```